docs/LearningOutcomes.adoc

```diff
@@ -20,7 +20,7 @@ toc::[]
 
 === References
 
-* https://se-edu.github.io/se-book/oopDesign/objects/encapsulation/[se-edu/se-book: Design: OOP: Objects: Encapsulation]
+* https://se-edu.github.io/se-book/oop/objects/encapsulation/[se-edu/se-book: Design: OOP: Objects: Encapsulation]
```

Comment:
In the link text, can you replace `Design` --> `Paradigms`
Please repeat the same in other places as well.